scripts/test_nearlib.sh

```diff
@@ -16,19 +16,23 @@ git clone https://github.com/near/create-near-app.git
 # Make sure to use local nearcore for tests
 export NODE_ENV=local
 
+SRC_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
@ailisp hard to grasp what this is trying to do. Why not just `SRC_DIR=$(pwd)/../`?

Reply:
BASH_SOURCE[0] would work in any director, `$(pwd)/../` wouldn't if current dir is not at `scripts/`

Reply:
@ailisp I see, then it can be `SRC_DIR="$(pwd)/$(dirname "${BASH_SOURCE[0]}")"`?

Reply:
That looks clever! in some case it would leave `.` or `..` on trailling, but should be fine, i can always postfix it with a "/" and `ls workspace/./` or `ls workspace//` both works 👍 
```
workspace bo$ chmod +x a.sh 
workspace bo$ ./a.sh 
/Users/bo/workspace/.
workspace bo$ cd ..
$ workspace/a.sh 
/Users/bo/workspace
$ cd workspace/near-ops
near-ops bo$ ../a.sh 
/Users/bo/workspace/near-ops/..
```